src/PlusDataCollection/Interson/vtkPlusIntersonVideoSource.cxx

```diff
@@ -122,7 +122,7 @@ class vtkPlusIntersonVideoSource::vtkInternal
   }
 
   //----------------------------------------------------------------------------
-  PlusStatus vtkPlusIntersonVideoSource::vtkInternal::InitializeDIB(int imageSize[2])
+  PlusStatus vtkPlusIntersonVideoSource::vtkInternal::InitializeDIB(std::vector<int> imageSize)

```

Comment:
@lassoan Are we still coding const & or are we letting the compiler decide?

Reply:
const& would be more appropriate

We should also replace std::vector<int> with a typedef and the std::vector<int> with a fixed-size class. We might use either itkFixedArray or just redefine it in Plus to avoid ITK dependency in such a core type.

Reply:
I am very happy with std::array<type, size> as it is iterator compatible.

Reply:
Agreed. Such change may cause regressions, so we should do it after we released the new stable version (in a few weeks). I've added a ticket to track that task (#188).

Reply:
Alright, I've added const & to the function definition.